src/shared/bus-unit-util.c

```diff
@@ -2937,3 +2938,73 @@ int bus_service_manager_reload(sd_bus *bus) {
 
         return 0;
 }
+
+int unit_freezer_restore(const char *name, UnitFreezer *ret) {
```

Comment:
I don't understand the function name at all. Why not `unit_freezer_new`??

Also, I'd

Reply:
The idea is you use `freezer_freeze` to create a freezer, then `freezer_cancel` to "cancel it". After it's canceled, you can call `freezer_restore` to bring it back into being, so that it gets thawed later. That was the idea, anyway...

Reply:
> Also, I'd

Cut off message?